curvefs/src/mds/topology/topology_item.h

```diff
@@ -548,7 +550,7 @@ class CopySetInfo {
 
     uint64_t GetPartitionNum() const { return partitionNum_; }
 
-    void SetPartitionNum(u_int64_t number) { partitionNum_ = number; }
+    void SetPartitionNum(uint64_t number) { partitionNum_ = number; }
```

Comment:
The partitionNum_ field is needn't anymore, remove it and related functions.

Reply:
done